.buildkite/windows.ps1

```diff
@@ -0,0 +1,96 @@
+cd $env:WORK_DIR
+
+# Pre-built rocksdb for Haskell
+git.exe clone https://github.com/facebook/rocksdb.git --branch v4.13.5
+curl.exe -L 'https://s3.eu-central-1.amazonaws.com/ci-static/serokell-rocksdb-haskell-325427fc709183c8fdf777ad5ea09f8d92bf8585.zip' -o D:\Downloads\rocksdb.zip
+7z x D:\Downloads\rocksdb.zip
+
+# CSL-1509: After moving the 'cardano-sl' project itself into a separate folder ('lib/'), the 'cardano-text.exe' executable fails on AppVeyor CI.
+# After some investigation, it was discovered that this was because 'rocksdb.dll' has to be located in this folder as well, or else the test executable doesn't work.
+copy rocksdb.dll node
+copy rocksdb.dll lib
+copy rocksdb.dll wallet-new
+
+############################################################################
+# Prepare stack and build tools
+#
+
+Write-Output "--- Stack setup and install tools"
+
+stack.exe config --system-ghc set system-ghc --global true
+stack.exe path
+stack.exe exec -- ghc-pkg recache
+stack.exe --verbosity warn setup --no-reinstall
+# Install happy separately: https://github.com/commercialhaskell/stack/issues/3151#issuecomment-310642487. Also install cpphs because it's a build-tool and Stack can't figure out by itself that it should be installed
+stack.exe --verbosity warn install happy cpphs -j 2 --no-terminal --local-bin-path $env:SYSTEMROOT\system32 --extra-include-dirs="D:\OpenSSL-Win64-v102\include" --extra-lib-dirs="D:\OpenSSL-Win64-v102" --extra-include-dirs="D:\xz_extracted\include" --extra-lib-dirs="D:\xz_extracted\bin_x86-64" --extra-include-dirs="$env:WORK_DIR\rocksdb\include" --extra-lib-dirs="$env:WORK_DIR"
+
+############################################################################
+# Build and test.
+#
+# We intentionally don't build auxx here, because this build is for installer.
+#
+
+Write-Output "+++ Stack build and test"
+
+stack.exe --dump-logs install cardano-sl cardano-sl-tools cardano-sl-wallet-new -j 3 --no-terminal --local-bin-path $env:WORK_DIR --no-haddock-deps --flag cardano-sl-core:-asserts --flag cardano-sl-tools:for-installer --extra-include-dirs="D:\OpenSSL-Win64-v102\include" --extra-lib-dirs="D:\OpenSSL-Win64-v102" --extra-include-dirs="D:\xz_extracted\include" --extra-lib-dirs="D:\xz_extracted\bin_x86-64" --extra-include-dirs="$env:WORK_DIR\rocksdb\include" --extra-lib-dirs="$env:WORK_DIR" --ghc-options="-copy-libs-when-linking"
+
+
+############################################################################
+# Assemble artifact zip file for use by the Daedalus installers build
+#
+
+Write-Output "--- Create zip file"
+
+# from here onwards, errors terminate the script
+$ErrorActionPreference = "Stop"
+
+#Cardano pieces, modulo the frontend
+mkdir daedalus
+## log config is called `log-config-prod.yaml` just in case, it's the old name
+copy log-configs\daedalus.yaml daedalus\log-config-prod.yaml
+copy lib\configuration.yaml daedalus\
+copy lib\*genesis*.json daedalus\
+copy cardano-launcher.exe daedalus\
```

Comment:
are we in `$env:WORK_DIR`? 
Can we do `copy $env:WORK_DIR\cardano-launcher.exe daedalus\`?

Reply:
Yes we are in `$env:WORK_DIR` but I don't want to touch this bit because it works and produces a zip with the expected layout.